### DISCUSSION TITLE
unregelmässiges Aufhängen der Raspberrymatic auf CCU3

### DISCUSSION THREAD

**Discussion by uwiuke** [OpenCCU/OpenCCU#1323] [🇩🇪 Fragen&Antworten]
... die CCU3 „hängt“ sich (nach zuvor wochenlangem problemlosen Agieren) regelmässig alle 1-4 Tage auf, sodaß auch ein Zugang nicht mehr möglich ist (WebUI).
Programme und restliche Einstellungen werden seit Wochen nicht mehr angeführt (also unveränderter Zustand).

...im Moment kann ich keine Reproduzierung ausmachen
...was kann ich tun, damit ich die Fehlerursache erkennen kann; die LOG-Datei habe ich angehängt

System information:
RaspberryMatic Version: ...3.57.5.20210525
Hardware: ...CCU3 von ElV
Keine HomeMatic RF-Module
Logdatei:
[ccu3-webui-2021-06-11.log](https://github.com/jens-maus/RaspberryMatic/files/6713919/ccu3-webui-2021-06-11.log)

**Comment by jens-maus** [comment]
> ... die CCU3 „hängt“ sich (nach zuvor wochenlangem problemlosen Agieren) regelmässig alle 1-4 Tage auf, sodaß auch ein Zugang nicht mehr möglich ist (WebUI).

Die erste Frage wäre, tritt das immer zur selben Uhrzeit z.B. auf? Auch sehe ich nicht wirklich ein Problem aus dem Logfile, ausser das da auch RedMatic installiert ist. Ich würde also hingehen und ein Add-on (inkl. RedMatic) nach dem anderen sukzessive abzuschalten bis das Problem dann über einen längeren Zeitraum nicht mehr auftritt. Und wenn "ein Zugang nicht mehr möglich ist"  müsste auch ersteinmal via ping&co überprüft werden ob die Zentrale wirklich komplett weg ist/stehengeblieben oder nur die WebUI nicht mehr läuft. Fragen über Fragen die am Ende nur du beantworten kannst.

**Comment by uwiuke** [comment]
… ich habe jetzt Redmatic komplett entfernt, das Problem tritt dennoch sporadisch auf, manchmal morgens, manchmal abends. Auf die WebUI kommt man noch drauf und man kann einen Neustart noch erzeugen, aber viel mehr geht nicht mehr.

Sobald man z.Bsp. die Geräteliste aufrufen möchte, oder die Programme, geht nichts mehr und die Webseite hängt sich auf. Dann heißt es nur noch: Browserfenster schließen, ca. 2 min warten und nochmals anmelden, dann kommt man wenigstens noch auf die Hauptseite und kann einen Neustart erzeugen.

Ich probiere jetzt aus, noch mehr Add-Ons auszuschalten und weiter zu beobachten.

 

VG Uwe

 

Von: Jens Maus ***@***.***>
Antworten an: jens-maus/RaspberryMatic ***@***.***>
Datum: Freitag, 25. Juni 2021 um 08:21
An: jens-maus/RaspberryMatic ***@***.***>
Cc: uwiuke ***@***.***>, Author ***@***.***>
Betreff: Re: [jens-maus/RaspberryMatic] unregelmässiges Aufhängen der Raspberrymatic auf CCU3 (#1323)

 

... die CCU3 „hängt“ sich (nach zuvor wochenlangem problemlosen Agieren) regelmässig alle 1-4 Tage auf, sodaß auch ein Zugang nicht mehr möglich ist (WebUI).

Die erste Frage wäre, tritt das immer zur selben Uhrzeit z.B. auf? Auch sehe ich nicht wirklich ein Problem aus dem Logfile, ausser das da auch RedMatic installiert ist. Ich würde also hingehen und ein Add-on (inkl. RedMatic) nach dem anderen sukzessive abzuschalten bis das Problem dann über einen längeren Zeitraum nicht mehr auftritt. Und wenn "ein Zugang nicht mehr möglich ist" müsste auch ersteinmal via ping&co überprüft werden ob die Zentrale wirklich komplett weg ist/stehengeblieben oder nur die WebUI nicht mehr läuft. Fragen über Fragen die am Ende nur du beantworten kannst.
